language_server/test/lsp/lsp_server_test.cpp

```diff
@@ -243,26 +243,4 @@ TEST(lsp_server, request_error_no_message)
     EXPECT_CALL(message_provider, reply(expected_telemetry));
 
     s.message_received(request_response);
-}
-
-TEST(lsp_server_test, wrong_message_received)
-{
-    test::ws_mngr_mock ws_mngr;
-    send_message_provider_mock smpm;
-    lsp::server s(ws_mngr);
-    s.set_send_message_provider(&smpm);
-
-    json expected_telemetry =
-        R"({"jsonrpc":"2.0","method":"telemetry/event","params":{
-            "measurements":null,
-            "method_name":"server_error/lsp_server/method_unknown_error",
-            "properties":null
-           }})"_json;
-
-    // Only telemetry expected
-
-    EXPECT_CALL(smpm, reply(expected_telemetry));
-
-    s.message_received(
-        R"({"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"user_storage:/user/storage/layout","languageId":"plaintext","version":4,"text":"sad"}}})"_json);
```

Comment:
If I remember correctly, the `user_storage` is a non-complaint URL schema and this test was verifying that we do not crash the server (anymore).

Reply:
Alright, I modified the test a little bit to ensure that the server doesn't crash on a non-compliant schema.